tests/pages/commandsUi.page.js

```diff
@@ -0,0 +1,51 @@
+
+exports.Commands = class Commands {
```

Comment:
Why the name Commands?
For page objects you should use a readable name of the page object that you want to model. Like Workspace.

Reply:
ok